backend/notifications/views.py

```
@@ -0,0 +1,49 @@
+from rest_framework import permissions, viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from . import services
+from .serializers import NotificationSerializer
+
+
+class NotificationViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = NotificationSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)